docs/howto/manage-relations.md

```diff
@@ -64,7 +60,7 @@ requires:
     scope: container
 ```
 
-Other than this, implement a subordinate relation in the same way as any other relation. Note however that subordinate units cannot see each other’s peer data.
+Other than this, implement a subordinate relation in the same way as any other relation. Note however that subordinate units cannot see each other's peer data.
 
 > See also: [Charm taxonomy](https://juju.is/docs/sdk/charm-taxonomy#heading--subordinate-charms)
```

Comment:
This page will be in Juju docs. We can live it as is for now (as in, there will  be a redirect that will take people where they need to be), but down the line we'll want to update all the links.

Reply:
Yeah, all I did here was change the curly quote to a straight quote, because that's how we've currently standardised on things in ops. Sphinx auto-curls them all later, so they can be straight in the plain files (at least this is what happens with the ReST, I'm assuming it does for the Markdown too).